### PR TITLE
Migrated bulk transfer and batch transaction API

### PR DESCRIPTION
**Pull request by dhruvsonagara**
## Description

- Migrated Camel API to Spring to enable swagger in it.
- Created separate interface and controller for API with same Endpoint.
- Updated Camel API with proper names to use it as function in Spring API.
- Added necessary detail of Request body, Response and Headers.
 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by dhruvsonagara** [comment]
@danishjamal104 & @fynmanoj Please review this PR. 
It's raised to unblock danish for his sprint
I'm not sure if this PR should be merged cause some api are still remaining to be migrated.

File: 1680070192969_ph-ee-bulk-demo-11.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id,request_id,payment_mode,payer_identifier_type,payer_identifier,payee_identifier_type,payee_identifier,amount,currency,note
```
Comment (danishjamal104) [inline 1680070192969_ph-ee-bulk-demo-11.csv:1]: Remove this file.

File: 1680260638925_null [hunk 2]
```diff
@@ -0,0 +1,6 @@
+id,request_id,payment_mode,payer_identifier_type,payer_identifier,payee_identifier_type,payee_identifier,amount,currency,note
```
Comment (danishjamal104) [inline 1680260638925_null:1]: This also.

File: 1680260684316_ph-ee-bulk-demo-11.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+id,request_id,payment_mode,payer_identifier_type,payer_identifier,payee_identifier_type,payee_identifier,amount,currency,note
```
Comment (danishjamal104) [inline 1680260684316_ph-ee-bulk-demo-11.csv:1]: This also.

File: ph-ee-bulk-demo-6.csv [hunk 4]
```diff
@@ -0,0 +1 @@
+src/test/java/resources/ph-ee-bulk-demo-6.csv
```
Comment (danishjamal104) [inline ph-ee-bulk-demo-6.csv:1]: Remove this also.

**Review by danishjamal104 — CHANGES_REQUESTED** [review]
Please remove the demo files from this PR. Rest looks good to me.

**Review by danishjamal104 — APPROVED** [review]
@fynmanoj please merge into dev branch

**Pull request merged**